pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetDsmParticipantStatusRoute.java

```diff
@@ -65,7 +74,10 @@ ParticipantStatusTrackingInfo process(String studyGuid, String userGuid, String
         LOG.info("DSM call completed, study={} and participant={}, status={}", studyGuid, userGuid, result.getStatusCode());
 
         if (result.getStatusCode() == 200) {
-            return new ParticipantStatusTrackingInfo(result.getBody(), status, userGuid);
+            // todo arz figure this part out
+            List<StudyWorkflowStatus> workflows = ElasticsearchServiceUtil.queryByStudyParticipant(esClient, umbrellaGuid, studyGuid, userGuid));
```

Comment:
Would be great to do a simple direct ES lookup of _only_ what we need here, and avoid pulling down a ton of stuff from the participant object.